packages/visualisation/src/Map.js

```diff
@@ -103,13 +104,14 @@ const Map = ({ cars, bookings, hubs, kommuner }) => {
     },
     getRadius: () => 3,
     // #fab
-    getFillColor: ({status}) => status === 'New' ? [255, 170, 187] : status === 'Delivered' ? [170, 187, 255] : [170, 255, 187, 0.3],
+    getFillColor: ({ status }) => status === 'New' ? [255, 170, 187] : status === 'Delivered' ? [170, 187, 255] : [170, 255, 187, 0.3],
     pickable: true,
-    onHover: ({object, x, y}) => {
+    onHover: ({ object, x, y }) => {
       if (!object) return setHoverInfo(null)
       setHoverInfo({
         type: 'booking',
-        title: object.address + (object.isCommercial ? ' (företag)' : ' (Status: ' + object.status + ')'),
+        title: object.address,
+        isCommercial: object.isCommercial ? '(företag)' : ' Status: ' + object.status,
```

Comment:
Is this correct? It was part of the title before

Reply:
I change it because I thought it was easier to read if it was not part of the title, but maybe title should be named address

Reply:
Oh wait I misunderstood the code a bit, I see the usage down below. Maybe `isCommercial` ought to be named something like `subtitle`, since it's not a boolean

Reply:
Fixed :)